### DISCUSSION TITLE
Blind App Quickstart with CRA-Nillion

### DISCUSSION THREAD

**Discussion by tofu9989** [NillionNetwork/discussions#78] [Show and tell]
### Show and tell project type

Other

### Github Repo Link

https://github.com/tofu9989/cra-nillion

### Project Description

I studied the code provided by the Nillion team to learn how to securely store and retrieve data using Nillion's decentralized technology. As a result, we built a CRA integration with the Nillion network.

### Is there anything else you want to share?

I have reviewed the project and believe it follows the guidelines correctly. Here is the transaction link as proof of deployment:

[https://testnet.nillion.explorers.guru/transaction/E44CB72D50B098127662B58A22B39B43B76C65BFFD66D866FFA56E4911C7E511?height=2552621](https://testnet.nillion.explorers.guru/transaction/E44CB72D50B098127662B58A22B39B43B76C65BFFD66D866FFA56E4911C7E511?height=2552621)

**Comment by Davetbutler** [comment]
Hey @tofu9989. Dave, from the product team here.

I see this is the second project you have submitted - that's awesome.

I wonder if you could answer the following two questions to help us make Nillion even better!

1. Where did you find out about Nillion?
2. What was your experience in getting started with Nillion, what did you try to build, and how did it go?

Feel free to reply here, or hit me up on Discord (`dave_97803`) if that is easier :)
